meta-leda-bsp/conf/machine/common-qemu-arm.inc

```diff
@@ -22,7 +23,7 @@ UBOOT_LOADADDRESS = "0x40008000"
 
 MACHINE_FEATURES = "usbgadget usbhost vfat"
 
-IMAGE_BOOT_FILES = "u-boot.${UBOOT_SUFFIX} ${KERNEL_IMAGETYPE}"
+IMAGE_BOOT_FILES = "u-boot.${UBOOT_SUFFIX} ${KERNEL_IMAGETYPE} boot.scr"
```

Comment:
There are variables UBOOT_ENV and UBOOT_ENV_SUFFIX for the "boot.scr" filename.
Not sure if they are available here, but I guess if so, they should be used.

Reply:
Moved UBOOT_ENV and UBOOT_ENV_SUFFIX to qemuarm machine config where they belong, recipes now use "${UBOOT_ENV}.${UBOOT_ENV_SUFFIX}"